examples/basics.carp

```diff
@@ -234,4 +234,5 @@
       (two-lengths-in-same-func)
       (changing-target-of-ref)
       (resolve-correctly)
+      0
```

Comment:
Is there a reason you want to explicitly return 0 here?

Reply:
Also curious about this